resources/build/test/test.sh

```diff
@@ -184,6 +192,14 @@ fi
 echo -e "${COLOR_BLUE}## Testing output of: builder_parse --feature xyzzy --bar abc --baz def test${COLOR_RESET}"
 parse_output=$(builder_parse --feature xyzzy --bar abc --baz def test)
 expected="$(builder_echo setmark "test.sh parameters: <--feature xyzzy --bar abc --baz def test>")"
+
+if [[ $VERSION_ENVIRONMENT == "local" ]]; then
+  expected="$(builder_echo setmark "test.sh parameters: <--feature xyzzy --bar abc --baz def test --debug>")"
+  # Yay, the fun of getting the second line... that newline before the closing brace is intentional.  :(
+  parse_output=${parse_output#*
+}
```

Comment:
Your PR comment explains why you are doing this -- it needs to be in the code!

```suggestion
if [[ $VERSION_ENVIRONMENT == "local" ]]; then
  expected="$(builder_echo setmark "test.sh parameters: <--feature xyzzy --bar abc --baz def test --debug>")"
  # Ignore the first line of the output message "Local build ... setting --debug"
  # Newline before the closing brace strips the new line and following text
  parse_output=${parse_output#*
}
```

However, I don't think it's even needed. You could use `$'\n'`:

```suggestion
if [[ $VERSION_ENVIRONMENT == "local" ]]; then
  expected="$(builder_echo setmark "test.sh parameters: <--feature xyzzy --bar abc --baz def test --debug>")"
  # Ignore the first line of the output message "Local build ... setting --debug"
  parse_output=${parse_output#*$'\n'}
```

Or actually test the new value, which is a little long admittedly, but is self-explanatory:

```suggestion
if [[ $VERSION_ENVIRONMENT == "local" ]]; then
  expected="$(builder_echo grey "Local build environment detected:  setting --debug")"$'\n'"$(builder_echo setmark "test.sh parameters: <--feature xyzzy --bar abc --baz def test --debug>")"
```